Tests/Unit/ContentObject/JsonDecoderTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FriendsOfTYPO3\Headless\Test\Unit\ContentObject;
+
+use FriendsOfTYPO3\Headless\ContentObject\JsonDecoder;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\TestingFramework\Core\Unit\UnitTestCase;
+
+class JsonDecoderTest extends UnitTestCase
+{
+    /**
+     * @param $testValue
+     * @param $expectedValue
+     *
+     * @test
+     * @dataProvider testValues
+     */
+    public function possibleFalsePositives($testValue, $expectedValue): void
+    {
+        $jsonDecoder = GeneralUtility::makeInstance(JsonDecoder::class);
+
+        self::assertSame($expectedValue, $jsonDecoder->isJson($testValue));
+    }
+
+    public function testValues(): array
```

Comment:
I would not use the test suffix here. Because it is not a test but a dataProvider

Reply:
yeah, good catch! :)